proto/rill/runtime/v1/catalog.proto

```diff
@@ -62,7 +62,7 @@ message MetricsView {
     string name = 1;
     string label = 2;
     string description = 3;
-    bool visible = 4;
+    bool ignore = 4;
```

Comment:
On reflection, the `ignore` doesn't really belong in the metrics view – it means you could end up with invalid dimensions/measures in the catalog, which the APIs are not currently set up to handle.

Probably the `ignore` should be handled already when parsing the YAML artifact?

Reply:
Makes sense.